safe-global/safe-wallet-web
# [Counterfactual] Display option to refresh balances

## What it solves

Resolves #

## How this PR fixes it

- Adds a section to the assets page for counterfactual safes to refresh balance and a link to block explorer
- The refresh button has a cooldown of 15s to mitigate potential spam

## How to test it

1. Create a counterfactual safe
2. Open the assets page
3. Observe a new section
4. Disconnect your wallet
5. Send ETH to your counterfactual safe
6. Wait until the transaction has been executed
7. Press "Refresh balance"
8. The new balance should be visible within the next 15s

## Screenshots
<img width="1259" alt="Screenshot 2024-02-08 at 21 20 47" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/4215d3e2-669c-4a44-aedc-901cc168c943">

## Checklist
